README.md

```diff
@@ -4,6 +4,20 @@ errcheck is a program for checking for unchecked errors in go programs.
 
 ![errcheck](https://github.com/kisielk/errcheck/workflows/errcheck/badge.svg)
 
+## Legacy Software
+This tool was written many years ago and has since been subsumed by a much more
+capable tool called ![staticcheck](https://github.com/dominikh/go-tools). It is
```

Comment:
This is technically not true. Staticcheck catches some error-related bugs, but it doesn't subsume errcheck. Staticcheck will never flag bare calls to functions that return errors, for example. nor are there equivalents for `-asserts` and `-blank`.

Reply:
I see... thanks! We can try to come up with a different message then.